### DISCUSSION TITLE
Automate the creation and localization of in-app purchases.

### DISCUSSION THREAD

**Discussion by Vlastelin1** [fastlane/fastlane#17960] [Q&A]
Is it real to automate the creation and localization of in-app purchases for google play console android with fastlane?

**Comment by max-ott** [comment]
`Fastlane` uses the Google Play Publishing API. I don't think the in-app purchase methods are implemented, but you can use them "manually". Check out the official docs: https://developers.google.com/android-publisher/api-ref/rest/v3/inappproducts
